src/components/ERC20Transfers/ERC20Transfers.tsx

```diff
@@ -0,0 +1,318 @@
+      },
+    },
+
+    {
+      title: "Sending Address",
+      dataIndex: "from_address",
+      key: "from_address",
+      render: () => {
+        if (!Array.isArray(fetchData)) {
+          return null;
+        }
+
+        return (
+          <div>
+            {fetchData.map((item) => {
+              const shortFromAddress = `${item.from.substring(
+                0,
+                6,
+              )}...${item.from.substring(item.from.length - 4)}`;
+
+              return (
+                <div>
+                  {/* <Blockie address={from} size={5} /> */}
+                  <p
+                    style={{
+                      fontSize: "1rem",
+                      color: "rgba(0, 0, 0, 0.6)",
+                      display: "inline-block",
+                      marginLeft: "5px",
+                    }}
+                  >
+                    {shortFromAddress}
+                  </p>
+                </div>
+              );
+            })}
+          </div>
+        );
+      },
+    },
+
+    {
+      title: "Receiving Address",
+      dataIndex: "to_address",
+      key: "to_address",
+      render: () => {
+        if (!Array.isArray(fetchData)) {
+          return null;
+        }
+
+        return (
+          <div>
+            {fetchData.map((item) => {
+              const shortTo =
+                item.to.substring(0, 6) +
+                "..." +
+                item.to.substring(item.to.length - 4);
+
+              return (
+                <div>
+                  {/* <Blockie address={from} size={5} /> */}
+                  <p
+                    style={{
+                      fontSize: "1rem",
+                      color: "rgba(0, 0, 0, 0.6)",
+                      display: "inline-block",
+                      marginLeft: "5px",
+                    }}
+                  >
+                    {shortTo}
+                  </p>
+                </div>
+              );
+            })}
+          </div>
+        );
+      },
+    },
+
+    {
+      title: "Hash",
+      dataIndex: "transaction_hash",
+      key: "transaction_hash",
+      render: () => {
+        // Make sure fetchData is defined and is an array
+        if (!Array.isArray(fetchData)) {
+          return null;
+        }
+
+        return (
+          <div>
+            {fetchData.map((item) => {
+              return (
+                <div style={{ display: "block" }}>
+                  <a
+                    href={`https://polygonscan.com/tx/${item.hash}`}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    Explorer
+                  </a>
+                </div>
+              );
+            })}
+          </div>
+        );
+      },
+    },
+  ];
+
+  return (
+    <div>
+      <div
+        style={{
+          margin: "0 1.5625rem 3.125rem 1.5625rem",
+          height: "100%",
+        }}
+      >
+        <Table
+          loading={!mockData}
```

Comment:
@karangorania, please introduce the loading animation just like in the ``ERC20Balance`` component.